src/Microsoft.AspNet.Http.Abstractions/project.json

```diff
@@ -1,11 +1,12 @@
 ﻿{
```

Comment:
Yikes, what is your auto-format set to? I thought VS preferred 2-space-tabs for json files.